pkg/cmd/propertiesGet.go

```diff
@@ -36,13 +36,15 @@ var (
 	// Variables set by cobra's command-line parsing.
 	propertiesPrefix       string
 	propertiesSuffix       string
+	propertiesInfix        string
 	propertiesOutputFormat string
 )
 
 func init() {
 	formatters := properties.GetFormatterNamesString(properties.CreateFormatters())
-	propertiesGetCmd.PersistentFlags().StringVar(&propertiesPrefix, "prefix", "", "the name of properties from a specified namespace with the provided prefix")
-	propertiesGetCmd.PersistentFlags().StringVar(&propertiesSuffix, "suffix", "", "the name of properties from a specified namespace with the provided suffix")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesPrefix, "prefix", "", "Prefix to match against the start of the property name within the namespace")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesSuffix, "suffix", "", "Suffix to match against the end of the property name within the namespace")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesInfix, "infix", "", "Infix(es) that could be part of the property name within the namespace, multiples are supplied as comma-separated list")
```

Comment:
Do you think something like "... multiple infixes can be supplied ..." would read better than "... multiples are supplied..."?